analyses/ID_cd_2010/03_sim_ID_cd_2010.R

```diff
@@ -0,0 +1,44 @@
+###############################################################################
+# Simulate plans for `ID_cd_2010`
+# © ALARM Project, March 2023
+###############################################################################
+
+# Run the simulation -----
+cli_process_start("Running simulations for {.pkg ID_cd_2010}")
+
+set.seed(2010)
+plans <- redist_smc(map,
+    nsims = 15e3,
+    runs = 2L,
+    counties = county_muni)
```

Comment:
This looks good. We just need to thin to 5,000 here by piping into:
```
 %>%
  group_by(chain) %>%
  filter(as.integer(draw) < min(as.integer(draw)) + 2500) %>% # thin samples
  ungroup()
  ```

Reply:
done!